### PR TITLE
Chronos: add built-in dataset: uci_electricity_wide

### PR DESCRIPTION
**Pull request by Rebekk**

### CODE REVIEW

File: python/chronos/src/bigdl/chronos/data/utils/public_dataset.py [hunk 1]
```diff
@@ -208,6 +211,26 @@ def preprocess_uci_electricity(self):
...
         return self
 
+    def preprocess_uci_electricity_wide(self):
+        '''
+        return data that meets the minimum requirements of tsdata.
+        '''
+        if not os.path.exists(self.final_file_path):
+            zip_file = zipfile.ZipFile(os.path.join(
+                                       os.path.expanduser(self.dir_path),
+                                       DATASET_NAME[self.name[:-5]][0]))
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/data/utils/public_dataset.py:221]: why we need this `[:-5]`

**Review by TheaperDeng — COMMENTED** [review]
This dataset is quite large so it might be unreasonable to have it in our automatic test. Please manually test it.

File: python/chronos/src/bigdl/chronos/data/utils/public_dataset.py (hunk 1 above)
Comment (Rebekk) [reply python/chronos/src/bigdl/chronos/data/utils/public_dataset.py:221]: Oh, I forgot I already add 'uci_electricity_wide' in DATASET_NAME. "[:-5]" was added for if we don't modify DATASET_NAME.

**Review by TheaperDeng — APPROVED** [review]
run jenkins

**Comment by Rebekk** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/807/

**Pull request merged**